### PR TITLE
User profile fully functional

### PR DESCRIPTION
**Pull request by wAndrewx**
### User Features
- Users can now access their profile data 
    - Hours Focused
    - Pomodoros completed
    - Days Logged in
- Update profile each cycle
- Refresh button to get up to date info

### CODE REVIEW

File: components/Navbar.js [hunk 1]
```diff
@@ -37,6 +43,7 @@ export const Navbar = () => {
         ? false
         : JSON.parse(window.localStorage.getItem('isLoggedIn'));
     setLoginBool(landingSession);
+    // window.addeventlistener
```
Comment (khoaHyh) [inline components/Navbar.js:46]: remove this

File: components/Pomodoro.js [hunk 2]
```diff
@@ -1,19 +1,18 @@
...
 
 const Pomodoro = () => {
   const [pomoTime, setPomoTime] = useState(1500);
   const [breakTime, setBreakTime] = useState(300);
   const [{ isPlaying }, setIsPlaying] = useState(false);
   const [sessionType, setSessionType] = useState(true); // session true = pomoclock , session false = breakclock
   const [timerPointer, setTimerPointer] = useState(pomoTime); // indicates which timer/session value to focus on
-
-  // console.log(pomoTime);
-  // console.log(breakTime);
```
Comment (khoaHyh) [inline components/Pomodoro.js:15]: remove these comments

File: components/authentication/login/StatusAlert.js [hunk 3]
```diff
@@ -12,21 +12,21 @@ import { useEffect, useRef, useState } from 'react';
 // Goal: Only updates on submit...
 // How: componentdidmount lifecycle...
 export const StatusAlert = ({ message, status }) => {
-  const [alertMessage, setAlertMessage] = useState('');
-  const [alertStatus, setAlertStatus] = useState('');
+  // const [alertMessage, setAlertMessage] = useState('');
+  // const [alertStatus, setAlertStatus] = useState('');
```
Comment (khoaHyh) [inline components/authentication/login/StatusAlert.js:16]: remove these

File: components/authentication/login/StatusAlert.js [hunk 4]
```diff
@@ -12,21 +12,21 @@ import { useEffect, useRef, useState } from 'react';
...
+  // useEffect(() => {
+  //   let isMounted = false;
+  //   if (!isMounted) {
+  //     setAlertMessage(message);
+  //     setAlertStatus(status);
+  //   }
+  //   return () => {
+  //     isMounted = true;
+  //   };
+  // }, []);
```
Comment (khoaHyh) [inline components/authentication/login/StatusAlert.js:27]: if these aren't needed, remove these

**Review by khoaHyh — CHANGES_REQUESTED** [review]
Apply changes that I left in comments

File: components/Pomodoro.js (hunk 2 above)
Comment (wAndrewx) [reply components/Pomodoro.js:15]: these are removed already

Comment (khoaHyh) [reply components/Pomodoro.js:15]: oops you're right

**Pull request merged**